client/src/containers/signup/SignupForm.js

```diff
@@ -179,16 +179,15 @@ class SignupForm extends Component {
                   error={validName === false}
                   name='name'
                   type='text'
-                  value={name}
-                  onBlur={this.handleBlur}
-                  onChange={this.usernameChange}/>
-                { userExists && errorDisplay &&
+                  onChange={this.handleChange}
+                  value={name}/>
+                {/* { userExists && errorDisplay &&
```

Comment:
I believe this is commented out. We should try to avoid having commented out code in our project to adhere to the Contribution Guidelines

Reply:
![Screenshot 2019-05-30 at 10 01 01 PM](https://user-images.githubusercontent.com/42834180/58648176-c1818e80-8326-11e9-8ce9-5b2d73b3b82a.png)

Reply:
working fine in my mac

Reply:
commented lines i have removed now

Reply:
@Akashkotian7 - so the error display issue is because of Firefox browser because I just checked on Google Chrome and I see what you're seeing.

Reply:
@mtn217  the issue is resolved now

Reply:
@Akashkotian7 - Awesome - I see it display now. However, it doesn't look exactly your screenshot. The error message shows up in two lines instead of one lines like your screenshot.
![pr-bug](https://user-images.githubusercontent.com/14936566/58724633-ff67cc80-8391-11e9-8a16-4aba8f221473.PNG)

Reply:
Also - remember to squash your commits.

Reply:
@mtn217  it is working fine in my mac (both google chrome and mozilla).I have made changes now can you please test it in your system and let me know the updates

Reply:
Ok it works for me now. Approving!